edge/pkg/edged/edged_pods.go

```diff
@@ -672,6 +673,45 @@ func (e *edged) makeEnvironmentVariables(pod *v1.Pod, container *v1.Container, p
 		mappingFunc = expansion.MappingFuncFor(tmpEnv)
 	)
 
+	// Env will override EnvFrom variables.
+	// Process EnvFrom first then allow Env to replace existing values.
+	for _, envFrom := range container.EnvFrom {
+		switch {
+		case envFrom.ConfigMapRef != nil:
+			cm := envFrom.ConfigMapRef
+			name := cm.Name
+			configMap, ok := configMaps[name]
+			if !ok {
+				if e.kubeClient == nil {
+					return result, fmt.Errorf("couldn't get configMap %v/%v, no kubeClient defined", pod.Namespace, name)
+				}
+				optional := cm.Optional != nil && *cm.Optional
+				configMap, err = e.configMapManager.GetConfigMap(pod.Namespace, name)
+				if err != nil {
+					if apierrors.IsNotFound(err) && optional {
+						// ignore error when marked optional
+						continue
+					}
+					return result, err
+				}
+				configMaps[name] = configMap
+			}
+
+			invalidKeys := []string{}
+			for k, v := range configMap.Data {
```

Comment:
Do we also need to add these lines here?
```go
if len(envFrom.Prefix) > 0 {
    k = envFrom.Prefix + k
}
```

Reply:
"Prefix" is used to prevent name conflicts when multiple ConfiMaps are referenced.
Because I don't see an example of this field in the configmap documentation of K8S, I understand that this is rarely used by others so I remove this codes. But It was like the previous behavior of removing envfrom, so I decided to add it, what do you think？

Reply:
Yes, let's add it back and keep consistent with K8s :)